cfpb/capital-framework-sandbox
# Bugfix for form fields

Fixed form field naming in newer components and added base form field selector before modifiers
## Changes
- Updated all `m-field` to `m-form-field`
- Added `m-form-field` base class to all modified fields.
## Testing
- run `npm run build && npm start` 
- check out cf-forms page
## Review
- @Scotchester 
- @KimberlyMunoz 

[Preview this PR without the whitespace changes](?w=0)
## Screenshots

**Before**

![screen shot 2016-10-05 at 11 54 18 am](https://cloud.githubusercontent.com/assets/1280430/19122977/76d9e744-8af2-11e6-9cc4-4626d01b3369.png)

**After**

![screen shot 2016-10-05 at 11 47 31 am](https://cloud.githubusercontent.com/assets/1280430/19122760/84105994-8af1-11e6-91ab-b923003b97ef.png)
## Checklist
